conf/defaults.yaml

```diff
@@ -285,3 +285,6 @@ pacemaker.kerberos.users: []
 #default storm daemon metrics reporter plugins
 storm.daemon.metrics.reporter.plugins:
      - "org.apache.storm.daemon.metrics.reporters.JmxPreparableReporter"
```

Comment:
Please refer this file:
https://github.com/apache/storm/blob/master/docs/storm-metrics-profiling-internal-actions.md
(btw, link for http://storm.apache.org/releases/1.0.0/storm-metrics-profiling-internal-actions.html is broken, and so 2.0.0-SNAPSHOT is.)

Codahale metrics is included as shaded so we don't guarantee other plugins are working properly.

We want to expose those metrics to consumer to store external services/storages, and where to store should be flexible by make it pluggable.